charts/prometheus-node-exporter/templates/_helpers.tpl

```diff
@@ -44,7 +44,7 @@ app.kubernetes.io/part-of: {{ include "prometheus-node-exporter.name" . }}
 app.kubernetes.io/version: {{ . | quote }}
 {{- end }}
 {{- with .Values.podLabels }}
```

Comment:
I have just noticed that having podLabels in here is wrong - `podLabels` are to be attached to pods only but through this helper get attached to all resources. Would you be willing to correct it? We could rename the field to `commonLabels`, keep it here and make the field `commonLabels` in values. `podLabels` should then be inserted in the daemonset pod template.

Reply:
Was out last week, will take a look today or tomorrow.